### ISSUE TITLE
Question- How to write Pact Body with empty values and how to assert on specific field of body

### ISSUE THREAD

**Issue by TheBsal** [pact-foundation/pact-net#206]
I am doing a POC on pact and i am trying to figure out few stuffs
Suppose my provider return model looks like below

public class TestModel
    {
        public string prop1 { get; set; }
        public string prop2{ get; set; }
        public string prop2{ get; set; }
        public int prop4    { get; set; }     
    } 

As a consumer i only care about prop1 and prop2 and its type and also don't care about the data which has been return.

So can i write my contract for that ? Something like below

{
  "consumer": {
    "name": "Cons"
  },
  "provider": {
    "name": "Pro"
  },
  "interactions": [
    {
      "description": "There is already an api"
      "providerState": "Upon Making a Get request",
      "request": {
        "method": "get",
        "path": "/v1/resource/city/status"
      },
      "response": {
        "status": 200,
        "headers": {
          "content-type": "application/json; charset=utf-8"
        },
		"body": {
			"prop1": "",
			"prop2:  ""
			
		}
      }
    }
  ],
  "metadata": {
    "pactSpecification": {
      "version": "2.0.0"
    }
  }
}

Also in the provider side when i perform a .Verify(), can i verify that certain properties are present? 
In above case, all my consumer cares about is those 2 properties are present and as a provider all i want to validate is i am returning those 2 properties.

**Comment by TheBsal** [comment]
Update on this. I was able to validate against the property on what consumer defines. However, is there a way for us to not care about the data ?

**Comment by neilcampbell** [comment]
Yep there sure is. You can use the Matcher feature https://docs.pact.io/getting_started/matching#pact-matching-features

Examples in PactNet: https://github.com/pact-foundation/pact-net/blob/master/Samples/EventApi/Consumer.Tests/EventsApiConsumerTests.cs#L289

You are probably looking for `Match.Type`
